ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/NettyPerfClient.java

```diff
@@ -80,14 +81,15 @@
 
   private final String host;
   private final int port;
-  private final String uri;
+  private final String path;
   private final int concurrency;
   private final long totalSize;
   private final byte[] chunk;
   private final SSLFactory sslFactory;
   private final String serviceId;
   private final String targetAccountName;
   private final String targetContainerName;
+  private final List<String[]> customizedHeaders = new ArrayList<>();
```

Comment:
reminder to change this to customHeaders

Reply:
This is also minor, but you could use the `Pair` class in ambry utils here since there are always 2 elements.